components-java-sdk/pom.xml

```diff
@@ -27,11 +27,38 @@
             <artifactId>components-java-sdk-autogen</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <!--This is our async framework. Dapr Java SDK uses the same -->
         <dependency>
             <groupId>io.projectreactor</groupId>
             <artifactId>reactor-core</artifactId>
             <version>3.3.11.RELEASE</version>
         </dependency>
-    </dependencies>
 
+        <!-- Netty's Unix Domain Socket support for Linux and Mac, for both AMD64 and ARM64 -->
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-transport-native-kqueue</artifactId>
+            <version>4.1.79.Final</version>
```

Comment:
This should be a variable to avoid repetition and to ensure this is kept in sync with the remaining Netty components.

Reply:
Fixed on 0615d0bf3f228b98728feb7dfae39f978181332d